### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cs-pranjali-munfan**
PMDB link : https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=22193

PR Title: Claroty - Added new actions

Description:
Added the following actions
  - Get Assets
  - Get Asset Details
  - Get Alerts
  - Get Alert Details
  - Get Tasks
  - Get Queries
  - Get Insights
  - Get Events

Impact:  All actions

Unit Testcase:
 - Install the Connector
 - Configure connector using valid credentials
 - Configure connector using invalid credentials
 - Check all the playbooks are present
 - Check all the playbooks tags are present
 - Check all the playbooks are in Info mode
- Test all actions with sample inputs
- Test Data ingestion configuration
- Test Data ingestion playbooks
- Verified data ingestion playbook execution flow

### CODE REVIEW

File: claroty/info.json [hunk 1]
```diff
@@ -0,0 +1,1169 @@
...
+            "count_in_page": "",
+            "count_filtered": ""
+          }
+        }
+      ],
+      "title": "Get Assets",
+      "description": "Get assets from claroty server based on input parameters you have provided.",
+      "enabled": true,
+      "parameters": [
+        {
```
Comment (cs-prerna-joshi) [inline claroty/info.json:189]: please take command filter like criticality, last seen etc as input parameters to the action and build query in operations.py. Also provide extra param for open query.

File: claroty/info.json [hunk 2]
```diff
@@ -0,0 +1,1169 @@
...
+        "count_total": "",
+        "count_in_page": "",
+        "count_filtered": ""
+      },
+      "title": "Get Alerts",
+      "description": "Get Alerts from claroty server based on input parameters you have provided.",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Filters",
```
Comment (cs-prerna-joshi) [inline claroty/info.json]: same as above.

**Comment by cs-suyog-jinturkar** [comment]
UTC: 
1. Check all the playbooks are in debug mode???? we should check for info mode right? 
2. What does it mean by following UTCs?
Data ingestion playbooks
Test data ingestion flow

@cs-pranjali-munfan

**Comment by cs-pranjali-munfan** [comment]
> UTC:
> 
> 1. Check all the playbooks are in debug mode???? we should check for info mode right?
> 2. What does it mean by following UTCs?
>    Data ingestion playbooks
>    Test data ingestion flow
> 
> @cs-pranjali-munfan

@cs-suyog-jinturkar  Updated the UTCs

**Comment by cs-pranjali-munfan** [comment]
@cs-prerna-joshi  added common filter parameters to 'Get Alerts' and 'Get Assets' action

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**